### DISCUSSION TITLE
GL-AR300M16 Support?

### DISCUSSION THREAD

**Discussion by atdx** [xchwarze/wifi-pineapple-cloner#29] [Q&A]
GL-AR300M16 is a variant of GL-AR300M without nand(128MB), can I install the GL-AR300M version on my GL-AR300M16?

![Screenshot 2024-06-22 152903](https://github.com/xchwarze/wifi-pineapple-cloner/assets/49784904/d8c6538c-8c70-41ad-98f2-fc4f9d32cd13)

**Comment by appservnetwork** [comment]
It's supported, but not worth it because of the small NAND (16MB).
Recommend GL.iNet AR750S https://www.aliexpress.com/item/1005007177894325.html
